CONTRIBUTING.md

```diff
@@ -41,12 +42,12 @@ Getting your contributions integrated is relatively straightforward, here
 is the checklist:
 
 * All tests pass
-* Consensus is reached. This basically mean that the reviewers add `LGTM` comments. `LGTM` is a
+* Consensus is reached. This basically means that the reviewers add `LGTM` comments. `LGTM` is a
```

Comment:
Good point. Initially, I purposely remained vague since I didn't want to write that consensus meant at least one 1 LGTM comment.

What about this change:
```
Consensus is reached. This basically means that the reviewers add `LGTM` comments.
```

with 

```
Consensus is reached. This usually means that at least one reviewer added a `LGTM` comment
and reasonable amount of time went without anyone objecting.
```